assets/js/components/form.js

```diff
@@ -124,5 +124,63 @@
          });
      }
  }
+
+/**
+ * Listen for submit events on forms and change the checkbox values
+ * to true/false values
+ * @return {void}
+ */
+const formCheckboxValReplace = () => {
+    // Get all forms on the page
+    const forms = document.querySelectorAll('form');
+
+    // If there are no forms for the current page
+    if (!forms.length) {
+        // Exit
+        return;
+    }
+
+    // Loop the forms
+    for (let i = 0; i < forms.length; i++) {
+        // Cache the current form
+        let form = forms[i];
+        // Get the form's checkboxes
+        const checkboxes = form.querySelectorAll('input[type="checkbox"]');
+        // If no checkboxes
+        if (!checkboxes.length) {
+            // Exit
+            return;
+        }
+
+        // Add submit listener to form
+        form.addEventListener('submit', (event) => {
+            // Prevent the form from submitting
+            event.preventDefault();
+            // Map the checkboxes to true/false values and re-submit
+            mapCheckboxBoolValues(event.target, checkboxes);
+        });
+    }
+};
+
+/**
+ * Map checkboxes values to bool values
+ * @param  {HTMLElement}        form
+ * @param  {Array<HTMLElement>} checkboxes
+ * @return {void}
+ */
+const mapCheckboxBoolValues = (form, checkboxes) => {
+    for (let i = 0; i < checkboxes.length; i++) {
+        let checkbox = checkboxes[i];
+
+        if (checkbox.checked) {
+            checkbox.value = true;
+        } else {
+            checkbox.checked = true;
```

Comment:
It might not be clear to someone else why you're setting an unchecked checkbox to `:checked`. Comments would help this function